### ISSUE TITLE
Knowledge base

### ISSUE THREAD

**Issue by OfficerPotato911** [eln2/eln2#212]
Implement some "for dummies" information.

ELN1 had some honestly lacking knowledge base about it. A lot of things didn't even have wiki entries. 

Also; not everyone knows everything about electricity! Maybe a book with some basic descriptions you might run into could be useful; such as defining just what all them terms like voltage, current, amperage, capacitance, inductance etc mean.

Descriptions should be added to blocks that aren't so straight forward - like a rheostat. WTH is a rheostat??

Basically; 
A wiki that you can rely a bit more on [Specifically with the stuff that's confusing], descriptions for objects that novice's might not fully understand, and quick definitions of electrical vocabulary.

Not necessarily a priority to be put at the top of the list at the moment but something to keep on the books.

**Comment by jrddunbr** [comment]
A bunch of this stuff I do plan to write cute little one-page PDF documents, akin kinda to what you might see in TIS-100 and various datasheets for electronic products. But, to be honest, a lot of this is also on Wikipedia already and they really cover a lot of topics far better than we ever could. This is really the case particularly for some complex signal processing pipelines where we should really only need to introduce that it exists and how to find more information.

Thanks for the feedback. I'll keep this open in case anyone has ideas/proposals on what they want to document, but at this stage in the game I don't think we have things worked out enough to bother documenting them - yet.

**Comment by jrddunbr** [comment]
Oh, also:

> ELN1 had some honestly lacking knowledge base about it. A lot of things didn't even have wiki entries.

You're welcome to ask @Baughn for Wiki access, the wiki is entirely populated by content from the people who play the game. While that wiki is only for Eln1 content, you are totally welcome to add more content there if you're still playing it.

It is worth noting that there are some differences between the Eln1 mod as you can download it on CurseForge, and the forked version that exists on my GitHub, so in some cases you will need to highlight the difference in the article.

**Comment by OfficerPotato911** [comment]
My github name doesn't match my other display name; I'm known as LargeFluffyGhostCat more broadly in the community of the mod and already contribute to the wiki! :)

I see the point in things already being explained in external sources; but in-game information is much more convenient!

**Comment by jrddunbr** [comment]
Yeah, we want to use [Patchouli](https://modrinth.com/mod/patchouli)

**Comment by Caeleron** [comment]
ELN1 did actually have a wiki, online and in-game, it's just that a lot of it was incomplete or buggy. The rest I was going to say Jrd beat me to. 

I would suggest playing around with this in the mean time: https://falstad.com/circuit/
Messing around is a great way to learn, and their example circuits teach a bunch of concepts.

**Comment by jrddunbr** [comment]
> ELN1 did actually have a wiki, online and in-game, it's just that a lot of it was incomplete or buggy.

~~Actually, only the CurseForge version. I deleted it from the fork relatively early on IIRC.~~

Edit: ~~I don't think I ever merged it.~~

Edit Edit: I did! It was just into [EAU](https://github.com/jrddunbr/EAU)

**Comment by DarthSidiousPalpatine** [comment]
Huh, maybe you can place a whole electricity paragraph of physics school book to teach people. With "experiments" paragraph.

**Comment by jrddunbr** [comment]
There's probably some legal implications to that, but one could buy one of the great RadioShack books from [Forest E. Mims.](https://en.wikipedia.org/wiki/Forrest_Mims)

eg. [Getting Started in Electronics: A Complete Electronics Course in 128 Pages!](https://www.amazon.com/Getting-Started-Electronics-Complete-62-5004/dp/B000NXEHYO/ref=tmm_pap_swatch_0?_encoding=UTF8&qid=&sr=) is a great book that I have on my bookshelf.
